pkg/metrics/metrics.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2023 SAP SE or an SAP affiliate company
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import "sort"
+
+const (
+	// LabelSucceeded is a metric label indicating whether associated metric
+	// series is for success or failure.
+	LabelSucceeded = "succeeded"
+	// ValueSucceededTrue is value True for metric label succeeded.
+	ValueSucceededTrue = "true"
+	// ValueSucceededFalse is value False for metric label failed.
+	ValueSucceededFalse = "false"
+
+	// ShootNamespace is the label for prometheus metrics to indicate shoot namespace
+	ShootNamespace = "shoot_namespace"
```

Comment:
Please make this change since druid must ideally not have any knowledge of whether it's running within Gardener control plane for shoots, or just for any etcds. 
 
```suggestion
	EtcdNamespace = "etcd_namespace"
```